tests/core/vm/test_spurious_dragon_computation.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+
+from eth_utils import (
+    to_canonical_address,
+)
+
+from eth.vm.message import (
+    Message,
+)
+
+from eth.vm.forks.spurious_dragon.computation import (
+    SpuriousDragonComputation,
+)
+
+from eth.vm.transaction_context import (
+    BaseTransactionContext,
+)
+
+
+NORMALIZED_ADDRESS_A = "0x0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6"
+NORMALIZED_ADDRESS_B = "0xcd1722f3947def4cf144679da39c4c32bdc35681"
+CANONICAL_ADDRESS_A = to_canonical_address("0x0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6")
+CANONICAL_ADDRESS_B = to_canonical_address("0xcd1722f3947def4cf144679da39c4c32bdc35681")
+CONTRACT_CODE_A = b""
+CONTRACT_CODE_B = b""
+CONTRACT_CODE_C = b""
```

Comment:
@pipermerriam Would like to get suggestion on what is the best way of having different kinds of contract code in our tests. Do we usually just hardcode the `bytecode` here?

Reply:
Typically yes.  I assume we don't need complex code and we could just do something like: `CODE = b'\x01' * 20000` or something to keep it concise.